### ISSUE TITLE
Refactor Wins page JS code

### ISSUE THREAD

**Issue by Aveline-art** [hackforla/website#1818]
### Overview
The script for the wins page could use a little bit of clean up. In the interest of maintainability and readability, the code may need a bit of revision.

Before starting, please see [this comment](https://github.com/hackforla/website/issues/1818#issuecomment-879535414).

### Action Items
- [x] Obtain an understanding of the JS code for the wins page and the way that code functions
  - [x] Take care to understand where inputs are coming from and outputs are going.
- [x] Add comments to lines that might be confusing, or require explanation
- [x] Conversely, remove comments that might not be required
- [x] Rename variables if the names are not intuitive
- [x] Condense code where doing so would improve clarity and conciseness
- [x] Do not break anything! The page should function the same way before and after
- [x] That said, if any bugs are noticed, quietly create a new issue for them (or tack it on here as an action item if you decide to tackle them with this issue). Templates are provided below.
  - [ ] Bug 1: [add bug here]
  - [ ] Bug 2: [add bug here]

### Resources/Instructions
This is a very freeform issue. It can be as big or as small as you want to be. The goal here is for future developers to discover this code, and be able to quickly understand the code and make changes. A large part of this is understanding our codebase and we encourage you to ask questions!

Some resources:
[Commenting](https://www.cs.utah.edu/~germain/PPS/Topics/commenting.html)
[Writing Comments in Python](https://realpython.com/python-comments-guide/)
[Mozilla's JavaScript Guide](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide)
[wins directory](https://github.com/hackforla/website/tree/wins-feature-1/pages/wins)

Note: Only wins-share-form.html and wins.html should have JS code.

**Comment by Aveline-art** [comment]
@akibrhast ping, testing

**Comment by Abel-Zambrano** [comment]
1. Started on it today
2. Working on issue #1596 
3. Weekdays after 5 pm PST
4. 5/17/2021

**Comment by jbubar** [comment]
IMPORTANT!! 
This issue is working off of a feature branch!
Please follow this wiki before creating a new branch!
[how to work off a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)

**Comment by Abel-Zambrano** [comment]
1. 70% complete
2. Time / Availablity
3. Weekdays after 5 pm PST
4. 7/28/2021
